data/state/addressConverters/plainAddressConverter.go

```diff
@@ -106,3 +108,45 @@ func (pac *PlainAddressConverter) PrepareAddressBytes(addressBytes []byte) ([]by
 func (pac *PlainAddressConverter) AddressLen() int {
 	return pac.addressLen
 }
+
+//the bech32 prefix will be erd, hrp separator is always 1
+//addresses will look like this erd1ajkdna0mnj56qwl4avp0qpmgc664x4qrdjq3rs6ddpsyfj9gna3s4sjy2n
+
+const erd = "erd"
+
+// ConvertToBech32 returns the address in bech32 format
+func (pac *PlainAddressConverter) ConvertToBech32(addressContainer state.AddressContainer) (string, error) {
+	if addressContainer == nil {
+		return "", state.ErrNilAddressContainer
+	}
+
+	conv, err := bech32.ConvertBits(addressContainer.Bytes(), 8, 5, true)
+	if err == nil {
```

Comment:
change to verify if err != nil then return error. like in createaddressFromBech32

Reply:
done